[Framework] Added support for items to parse in mapping (#429)
port-labs/ocean: docs/framework-guides/docs/framework/features/resource-mapping.md

| `createMissingRelatedEntities`          | bool  | `false` | Create missing related entities when the child entity is created.                                                                                                                                                               |
| `resources`                             | array | `[]`    | A list of resources to map.                                                                                                                                                                                                     |
| `resources.[].kind`\*                   | str   |         | The kind name of the resource. (Should match one of the available kinds in the [integration specification](../../develop-an-integration/integration-spec-and-default-resources.md#features---integration-feature-specification)) |
| `resources.[].selector.query`\*         | str   |         | A JQ expression that will be used to filter the raw data from the 3rd-party application.                                                                                                                                        |

> | `resources.[].port.itemsToParse`        | str   |         | A JQ expression that will be used to extract multiple entities and apply the mapping on instead of applying it on a single entity.                                                                                              |

Reviewer: ```suggestion
| `resources.[].port.itemsToParse`        | str   |         | A JQ expression that will be used to apply the mapping on the items of an array and generate multiple entities from the array items.                                                                                              |
```
